sdk/digitaltwins/digitaltwins/README.md

```diff
@@ -83,22 +94,59 @@ See https://github.com/Azure/ms-rest-browserauth to learn how to authenticate to
         const dependenciesFor = ["testdependenciesFor"];
         const includeModelDefinition = true;
         const maxItemCount = 1;
-        client.digitalTwinModels.list(dependenciesFor, includeModelDefinition, maxItemCount).then((result) => {
-          console.log("The result is:");
-          console.log(result);
-        }).catch((err) => {
-          console.log("An error occurred:");
-          console.error(err);
-        });
+        client.digitalTwinModels
+          .list(dependenciesFor, includeModelDefinition, maxItemCount)
+          .then((result) => {
+            console.log("The result is:");
+            console.log(result);
+          })
+          .catch((err) => {
+            console.log("An error occurred:");
+            console.error(err);
+          });
       });
     </script>
   </head>
   <body></body>
-</html>
+</html
+```
+
+## Key concepts
+
+Azure Digital Twins Preview is an Azure IoT service that creates comprehensive models of the physical environment. It can create spatial intelligence graphs to model the relationships and interactions between people, spaces, and devices.
+You can learn more about Azure Digital Twins by visiting [Azure Digital Twins Documentation](https://docs.microsoft.com/en-us/azure/digital-twins/)
+
+## Examples
```

Comment:
sample code above should be moved down here to examples

Reply:
That would be in the next update